students/BrianNisonger/Lesson2/assignment/charges_calc.py

```diff
@@ -0,0 +1,125 @@
+#pylint: disable=W0703
+'''
+Returns total price paid for individual rentals
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+
+
+def parse_cmd_arguments():
+    """
+    Parses commands from the commandline
+    """
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('-i', '--input', help='input JSON file', required=True)
+    parser.add_argument(
+        '-o', '--output', help='ouput JSON file', required=True)
+    parser.add_argument(
+        '-d', '--debug', help='debugger level', required=False, default=0)
+    return parser.parse_args()
+
+
+def configure_logging(debug_level):
+    """
+    Configures the logging
+    """
+    format_logger = '%(asctime)s %(filename)s:%(lineno)-3d%(levelname)s'
+    format_logger += ' %(message)s'
+    log_file = datetime.datetime.now().strftime('%Y-%m-%d') + '.log'
+    debug_level = int(debug_level)
+    if debug_level == 0:
+        logging.basicConfig(level=logging.CRITICAL)
+    if debug_level > 0:
+        logging.basicConfig(format=format_logger, level=logging.ERROR)
+        f_handler = logging.FileHandler(log_file)
+        f_handler.setLevel(logging.WARNING)
+        logging.getLogger().addHandler(f_handler)
+    if debug_level == 2:
+        logging.getLogger().setLevel(logging.WARNING)
+    if debug_level == 3:
+        logging.getLogger().setLevel(logging.DEBUG)
+
+
+def load_rentals_file(filename):
+    """
+    Loads json file of rental data
+    """
+    logging.debug("load_rentals_file")
+    logging.debug("Opens a json file and reads it in")
+    with open(filename) as file:
+        try:
+            data1 = json.load(file)
+        except Exception as error_msg:
+            logging.error(error_msg)
+            data1 = None
+    return data1
+
+
+def calculate_additional_fields(data_new):
+    """
+    Calculates additional fields from raw property data
+    """
+    logging.debug("calculate_additional_fields")
+    if data_new:
+        for value in data_new.values():
+            try:
+                warning_string = f"Incorrect date for {value['product_code']}"
```

Comment:
This is a great idea. We often put strings like this in a config so they can be re-used lots of places, and checked in tests.